scripts/backfill_cdc.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Backfills all the CDC tables
+set -e
+
+declare -a STORAGES=("groupedmessages"  "groupassignees")
+declare -a TABLES=("groupedmessage_local" "groupassignee_local")
```

Comment:
Is this script meant to only be run in the dev environment? Is that why the tables are local?

Reply:
Yes. this is indeed meant only for the development enviornment. 
For prod, it may look similar but I doubt we would do that in one step only. 
For self hosted the tables are the same.

Reply:
I would say we can make it a parameter to the script as the user could be running this from a place outside snuba. Worst case scenario if they pick the wrong setting the script will fail

Reply:
Parameter to the script sounds fine too. I suggested environment variable because I was thinking we could later hook into the same variable to launch the Snuba devservice in either local or dist mode automatically depending on its value. This is assuming the place it'll be called from is Sentry.

Reply:
I removed the parameter from the script entirely. https://github.com/getsentry/snuba/pull/1860
In dev there is no reason to specify that table or to expose the clickhouse table name.